introduction.md

```diff
@@ -1 +1 @@
-I am Nikesh Bhawsar an tech enthusiast.
My favorite Code School path is JavaScript and iOS.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?